File: Data Structures & Algorithms/two-integer-sum/submission-2.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> numValue = new HashMap<>();
        for(int i=0; i< nums.length;i++){
            int value =target - nums[i];
            if (numValue.containsKey(value)){
                return new int[]{numValue.get(value),i};
            }
            numValue.put(nums[i],i);

        }
        return new int[] {};
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
      
        max_length = 0

        for i in nums:
            if i-1 not in nums:
                j =i
                length =1
                while j+1 in nums:
                   
                    length +=1
                    j=j+1
                max_length = max(max_length,length)
        return max_length

            

        